rust-actix/.dockerignore

```diff
@@ -1 +0,0 @@
-/target
```

Comment:
Why is this port of the PR?

Reply:
eduardo told me to delete them. They should never have been merged since they fuck with the makefile as they are ready yet